backend/app/Main.hs

```diff
@@ -4,7 +4,28 @@ module Main
 
 import Universum
 
-import Stub (stub)
+import qualified Data.Yaml as Y
+import Options.Applicative (execParser, fullDesc, helper, info, progDesc)
+import qualified Options.Applicative as Opt
+
+import Edna.Config (EdnaConfig)
+import Edna.Web.Server (runEdna)
+
+-- | CLI parser for config path.
+configPathParser :: Opt.Parser FilePath
+configPathParser = Opt.strOption $
+  Opt.short 'c' <>
+  Opt.long "config" <>
+  Opt.metavar "FILEPATH" <>
+  Opt.help "Path to configuration file."
+
 
 main :: IO ()
-main = print stub
+main = do
+  configPath <- execParser $
+    info (helper <*> configPathParser) $
+    fullDesc <> progDesc "Edna API server."
+
+  Y.decodeFileEither @EdnaConfig configPath >>= \case
+    Left e -> putText (show e)
```

Comment:
In this case we will exit with 0 code which is not what we want. Also the error message should probably go to stderr. Why not use `decodeFileThrow` which does the right thing we want here?